src/Features/Auth/Login/LoginForm/LoginForm.jsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from 'react';
+
+import FormComponent from '../../../../components/FormComponent';
+
+function LoginForm() {
+  const [loginForm, setLoginForm] = useState({});
```

Comment:
There should be default state for login form like ..
`{name: '', password: ''}`